doc/tutorials/content/sources/spherical_voxel_grid/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+
+project(spherical_voxel_grid)
+
+find_package(PCL 1.8.1.99 REQUIRED)
```

Comment:
Even though we know this will be 1.9.0, such version does not exist yet, current master compiles into 1.8.1.99. If we already request 1.9.0, it will be impossible to build this tutorial.

Reply:
This places on us the responsibility of greping everything before the release and bumping versions on all tutorials. But that's we fact what we commit to, as I can see in #1683.